### PR TITLE
Issue #3048: Enable password toggles on all core password fields.

### LINKED ISSUE #3048: [UX] Enable the password toggle on all core password fields
## Describe your issue or idea

Should there be a show password toggle on the Database configruation installation step?

39031192-9b408460-441b-11e8-9287-b1c8ac863c7f

### Steps to reproduce (if reporting a bug)

* Install Backdrop
* On the `Database configuration` step it asks for the Database password
* In #297 we added a password show/hide toggle fo the backdrop users password field (user 1)

### Actual behavior (if reporting a bug)

Currently the database password field does not have this toggle.

### Expected behavior (if reporting a bug)

Up for debate, but should we provide a consistent experience here and add a show/hide password toggle?

### Relevant version/system information (if applicable)

`1.x`

### PR DESCRIPTION
**Pull request by ghost**
Fixes https://github.com/backdrop/backdrop-issues/issues/3048

### CODE REVIEW

File: core/modules/user/user.module [hunk 1]
```diff
@@ -1631,6 +1632,7 @@ function user_login($form, &$form_state) {
   $form['pass'] = array('#type' => 'password',
```
Comment (klonos) [inline core/modules/user/user.module]: While at it, should we fix the wrapping here as well?:

```suggestion
  $form['pass'] = array(
    '#type' => 'password',
```

**Review by klonos — APPROVED** [review]
LGTM 👍

**Comment by ghost** [comment]
Thanks @klonos, good suggestion. I missed that.

**Comment by ghost** [comment]
Merged into 1.x and 1.16.x.

**Pull request merged**